.ci/gcb-run-rake-tests.yml

```diff
@@ -0,0 +1,9 @@
+---
+steps:
+- name: 'gcr.io/$PROJECT_ID/downstream-builder'
```

Comment:
Oh, this totally works but I found it a little surprising that we're using the purpose-built container for a different purpose, maybe a comment about how we can safely reuse this container would be helpful.

Reply:
I can add a comment in the downstream-builder qualifying it's usage, to ensure that one is aware of where else it's used. Is that what you were thinking?

I debated making a new container, but it seemed somewhat unnecessary since it's a small entrypoint switch.

Reply:
Yep, that's what I was thinking - agree that the new container is unnecessary.

Reply:
sounds good!